.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
There should already be a .env file in the backend directory. I don't think we need one in the root dir as well

Reply:
Mb I overlooked the .env listing in the gitignore located in "backend" folder

Reply:
PR was resolved by implementing the same data model in models.py